brooklyn-server/core/src/test/java/org/apache/brooklyn/util/core/task/DynamicSequentialTaskTest.java

```diff
@@ -88,7 +88,11 @@ public void setUp() {
     
     @AfterMethod(alwaysRun=true)
     public void tearDown() throws Exception {
-        if (em != null) em.shutdownNow();
+        if (em != null) {
+            // need to await termination, otherwise interrupted-but-still-running threads 
+            // may update the cancellations/messages and interfere with subsequent tests
+            Assert.assertTrue(em.shutdownNow(Duration.FIVE_SECONDS));
```

Comment:
Personal preference to use `Asserts.DEFAULT_TIMEOUT` pretty much everywhere, unless a test is genuinely time-sensitive. I don't see any reason to decide on different length of time to wait when doing functional testing (i.e. when waiting for something to complete asyc).